### ISSUE TITLE
Breaks with WPGraphQL v1.13+

### ISSUE THREAD

**Issue by Ririshi** [harness-software/wp-graphql-gravity-forms#334]
### Description

An update in WPGraphQL v1.13, causing schema conflicts when connections are registered to GraphQL types that do not implement `Node`. The Gravity Forms plugin does this with `FormField`, causing the following errors payload in the GraphiQL IDE when the Gravity Forms plugin is activated next to WPGQL v1.13+:

```json
{
  "errors": [
    "Interface field Connection.nodes expects type [Node!]! but GfFormToFormFieldConnection.nodes is type [FormField!]!.",
    "Interface field Connection.nodes expects type [Node!]! but FormFieldConnection.nodes is type [FormField!]!.",
    "Interface field Edge.node expects type Node! but GfFormToFormFieldConnectionEdge.node is type FormField!.",
    "Interface field FormFieldConnectionEdge.node expects type Node! but GfFormToFormFieldConnectionEdge.node is type FormField!.",
    "Interface field Connection.nodes expects type [Node!]! but GfEntryToFormFieldConnection.nodes is type [FormField!]!.",
    "Interface field Edge.node expects type Node! but GfEntryToFormFieldConnectionEdge.node is type FormField!.",
    "Interface field FormFieldConnectionEdge.node expects type Node! but GfEntryToFormFieldConnectionEdge.node is type FormField!."
  ]
}
```

### Steps to reproduce

1. Install and activate WPGraphQL v1.13+
2. Install and activate WP GraphQL Gravity Forms v0.11.8
3. Open the GraphiQL IDE in the WordPress admin.

### Additional context

Related issues on WPGraphQL and the WooCommerce plugin:
https://github.com/wp-graphql/wp-graphql/issues/2629
https://github.com/wp-graphql/wp-graphql-woocommerce/issues/673

### Plugin Version

v0.11.8

### Gravity Forms Version

2.6.8.1

### WordPress Version

6.1.1

### WPGraphQL Version

v1.13.4

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have disabled ALL plugins except for Gravity Forms, WPGraphQL, and WPGraphQL for Gravity Forms

- [X] Yes
- [ ] My issue is with a specific 3rd-party plugin.

**Comment by justlevine** [comment]
Thanks for the heads up @Ririshi ! 
I've identified the culprit and am hoping to get a solve pushed ASAP.

In the interim, here's the official link to downgrade back to [WPGraphQL v1.12.3](https://github.com/wp-graphql/wp-graphql/releases/tag/v1.12.3)

**Comment by justlevine** [comment]
Fixed in 0.11.9 https://github.com/harness-software/wp-graphql-gravity-forms/releases/tag/v0.11.9
